adam-core/src/test/scala/org/bdgenomics/adam/rdd/feature/FeatureRDDSuite.scala

```diff
@@ -732,5 +732,56 @@ class FeatureRDDSuite extends ADAMFunSuite with TypeCheckedTripleEquals {
     assert(c.filter(_._1.isEmpty).forall(_._2.size == 1))
     assert(c0.filter(_._1.isEmpty).forall(_._2.size == 1))
   }
-}
 
+  sparkTest("don't lose any features when piping as BED format") {
+    val inputPath = testFile("dvl1.200.bed")
+    val frdd = sc.loadBed(inputPath)
+    val features = frdd.rdd.count
+
+    implicit val tFormatter = BEDInFormatter
+    implicit val uFormatter = new BEDOutFormatter
+
+    val pipedRdd: FeatureRDD = frdd.pipe("tee /dev/null")
```

Comment:
It was the easiest one-liner I could think of that would pipe standard in to standard out unmodified without creating any artifacts.